source/agora/common/EnrollmentManager.d

```diff
@@ -205,6 +205,68 @@ public class EnrollmentManager
         this.db.execute("DELETE FROM validator_set WHERE key = ?", enroll_hash[]);
     }
 
+    /***************************************************************************
+
+        In validatorSet DB, return the enrolled block height.
+
+        Params:
+            enroll_hash = key for an enrollment block height
+
+        Returns:
+            the enrolled block height.
+
+    ***************************************************************************/
+
+    public size_t getEnrolledHeight (const ref Hash enroll_hash) @safe
+    {
+        return () @trusted {
```

Comment:
And if the enrollment doesn't exist, what is the return value?

Reply:
Surely this creates 'no rows available' msg when there are keys don't match.
This is not an error, so we will correct it to return zero.

Reply:
Could you add a test-case?

Reply:
Sure, added test-case code.